layouts/updates/list.html

```diff
@@ -9,9 +9,9 @@
     <th>Update File Download</th>
     <th>Image Download</th>
   </tr>
-  {{ range .Pages.ByDate.Reverse }}
+  {{ range $index, $_ := .Pages.ByDate.Reverse }}
     <tr>
-      <td><a href='{{ .RelPermalink }}' title="See a summary of the changes in this release">{{ .Title }}</a></td>
+      <td><a href='{{ .RelPermalink }}' title="See a summary of the changes in this release">{{ .Title }}</a>{{ if (eq $index 0) }}(Latest){{ end }}</td>
```

Comment:
There is a space missing between the update name and (Latest).

Reply:
I also think it could be nice to have the text (Latest) in a different colour to highlight it.

Reply:
Fixed in d3101fb, I think colouring it might be a bit tacky.